app/controllers/users_controller.rb

```diff
@@ -1,3 +1,28 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
+  end 
+
+  def new
+    @user = User.new
+  end
+
+  def create
+  @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+  # Handle a successful save.
+  else
+    render 'new'
```

Comment:
```suggestion
    render :new
``` and add message for this case